src/webgpu/util/f32_interval.ts

```diff
@@ -85,15 +73,15 @@ export class F32Interval {
     return `[${this.begin}, ${this.end}]`;
   }
 
-  /** @returns a singleton for the infinite interval
+  /** @returns a singleton for the undefined interval
    * This interval is used in situations where accuracy is not defined, so any
    * result is valid.
    */
-  public static infinite(): F32Interval {
-    if (this._infinite === undefined) {
-      this._infinite = new F32Interval(Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY);
+  public static undefined(): F32Interval {
```

Comment:
A little bit troubling that this may collide with [Javascript's `undefined` property](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined).
This is the sort of thing that I'd expect to go horribly and subtly wrong with different JS engines, or emission with a transpiler like TypeScript.
Suggest renaming (`any()`?)